Stripe/STPTheme.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  STPTheme.swift
+//  Stripe
+//
+//  Created by Jack Flintermann on 5/3/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+import UIKit
+
+/// STPTheme objects can be used to visually style Stripe-provided UI. See https://stripe.com/docs/mobile/ios/standard#theming for more information.
+public class STPTheme: NSObject {
+
+  /// The default theme used by all Stripe UI. All themable UI classes, such as `STPAddCardViewController`, have one initializer that takes a `theme` and one that does not. If you use the one that does not, the default theme will be used to customize that view controller's appearance.
+  @objc public static let defaultTheme = STPTheme()
+
+  /// The primary background color of the theme. This will be used as the `backgroundColor` for any views with this theme.
+  @objc public var primaryBackgroundColor: UIColor = STPThemeDefaultPrimaryBackgroundColor
```

Comment:
I guess we're ditching the `null_resettable` behavior for this property? Is there a new approach to resetting, if someone was relying on that behavior?

Reply:
Yeah, we decided to make these nonnull instead, as we couldn't think of a reason someone would want to use them in a null_resettable way. If someone wants to reset to the default value, they can create an STPTheme() and get the initial values from there.